examples/example_obj.js

```diff
@@ -1 +1,50 @@
+class Cup {
+    //3 properties. talpa, spalva, currentFill. 2 methods : atsigerti(kiek isgerti) , pripilti(kiek pripilti). Privatus props ir getteriai setteriai
+    //if atsigerti > current fill, error = bandoma atsigerti per daug
+    constructor(talpa, spalva, currentFill){
+        this._talpa = talpa;
+        this._spalva = spalva;
+        this._currentFill = currentFill;
+    }
 
+    get currentFill() {
+        return this._currentFill;
+    }
+
+    set currentFill(currentFill) {
+        if (this.talpa < currentFill) {
+            throw new Error("cup is overflowing")
+        }
+       this._currentFill = currentFill;
+    }
+    get talpa(){
+        return this._talpa;
+    }
+    set talpa(talpa){
+        this._talpa = talpa;
+    }
+    atsigerti(kiek_isgerti){
+        if(this.currentFill < kiek_isgerti){
```

Comment:
Fix formatting.
If you get tired of this, install Prettier plugin in VS Code. Look for one with most downloads. Leave defaults and set up to format on save.